.mci.yml

```diff
@@ -116,7 +117,20 @@ tasks:
                   make
                   ${test_params} ./src/bsoncxx/test/test_bson
                   ${test_params} ./src/mongocxx/test/test_driver
-                  make run-examples
+                  ${test_params} ./examples/bsoncxx/builder_basic
```

Comment:
Actually, this should be a `find` with `xargs` so that we do not have update .mci.yml if new examples are added.